science/propka/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               github 1.0
+
+name                    propka
+revision                0
+
+supported_archs         noarch
+platforms               {darwin any}
+license                 LGPL-2.1
+maintainers             {gmail.com:howarth.at.macports @jwhowarth} openmaintainer
+
+description             Heuristic pKa calculations with ligands
+long_description        {*}${description}
+
+# The PyPi copy of propka 2.0.1 has a broken testsuite so
+# use the Jensen Group's copy from GitHub.
+
+github.setup            jensengroup propka 3.5.1 v
```

Comment:
traditionally, the `github.setup` field goes at the top of the portfile just after the PortGroups. Also, the `revision` field should ideally be close-by here...